pages/common/chars.md

```diff
@@ -0,0 +1,20 @@
+# chars
+
+> Use this tool to display names and codes for various ASCII (and unicode) characters / code points.
+> More information: <https://github.com/antifuchs/chars>.
+
+- Look up a character by its face value:
```

Comment:
```suggestion
- Look up a character by its value:
```
I don't think the word "face" adds anything to the sentence here. Does it have a special meaning that I'm missing? If not, I'd suggest removing it.

Reply:
It's just stressing the fact that you can enter the value into the program as you **see** it.

I will remove the "face" part